src/SFA.DAS.Payments.RequiredPayments.Messages/SFA.DAS.Payments.RequiredPayments.Messages.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Would it be best to leave the .Messages projects that are published to Nuget packages as .net standard, to allow maximum interoperability with other frameworks?

Reply:
That would probably be an @owencampbell or @abroodie decision.